include/rtc/description.hpp

```diff
@@ -71,6 +71,8 @@ class Description {
 		Direction direction() const { return mDirection; }
 		void setDirection(Direction dir);
 
+		void addSSRC(uint32_t ssrc, std::string name);
```

Comment:
Actually `Entry` is the common class for application m-line and media m-line to help factoring what can be in the SDP parsing and generation, so media-only properties would be better in `Media`.